CodeStarterCamp_Week2/main.swift

```diff
@@ -42,3 +42,45 @@ func checkingNumbers(){
 }
 
 checkingNumbers()
+
+/*
+ 1. 회차와 Lotto 번호들을 저장하는 Dictionary 타입의 변수 생성
+ - "1회차" : [1, 2, 3, 4, 5, 6] 과 같은 Key와 Value 값을 가짐
+ 2. Step 2 의 로또 당첨 번호 생성 함수를 호출할 때 마다, 회차와 로또 번호를 Dictionary에 저장
+ - 이를 구현하기 위해 별도의 함수 생성
+ - 새로운 로또 당첨 번호를 생성하면 직전 회차에 1을 더해 금번 회차를 만들어 냄
+ - 회차를 Key로, 새로운 로또 당첨 번호를 Value로 갖는 Dictionary에 저장
+ 3. 로또 당첨 번호를 생성한 후 Dictionary에 저장된 로또 당첨 번호를 확인
+ - 로또 당첨 번호를 5번 생성
+ - Dictionary에서 원하는 회차의 로또 당첨 번호를 찾음 (Optional로 안전하게 값 추출)
+ - Dictionary에서 추출한 원하는 회차의 로또 당첨 번호를 아래와 같이 출력
+      2회차의 로또 당첨 번호는 1, 2, 3, 4, 5, 6 입니다
+ */
+
+// 빈 Dictionary 생성
+var lottoResults: [String: [Int]] = [:]
+
+// Step2 를 실행하면서 Dictionary 를 채우는 함수
+func generateLottoResults() {
+    let winningNumbers = makeWinningNumbers()
+    let currentRound = (lottoResults.count + 1) // 현재회차는 직전회차 + 1
+    let roundKey = "\(currentRound)회차" // String 변수로 할당
```

Comment:
```suggestion
    let roundKey = "\(lottoResults.count + 1)회차"
```
불필요한 변수를 줄일 수 있을 거 같네요!